instagram/src/components/Login/Login.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+import './Login.css';
 
-class Login extends Component {
+class Login extends React.Component {
```

Comment:
Here, you could done one of two things. Add `{ Component }` to your `'react'` import or add `React` on line 4 as you have done here.